docs/api-reference/porcelain/application.rst

```diff
@@ -17,7 +17,15 @@ object on each edit. This model is identical to to Kubernetes *Deployment*
 objects and their ``.spec.template`` field, which serves as a mold for
 *ReplicaSet* objects (and by extension, *Pod* objects).
 
-The ``.spec.template`` field will be copied to a new *Release*
+Application's ``.spec.template.chart`` contains ambiguity by design: a user is
+expected to provide either a specific chart version or a *SemVer constraint*
+defining the range of acceptable chart versions. Shipper will resolve an
+appropriate available chart version and pin the *Release* on it. Shipper
+resolves the version in-place: it will substitute the initial constraint with a
+specific resolved version and preserve the initial constraint in the Application
+annotation.
```

Comment:
Would be nice to specify which annotation.

Reply:
Pushed a fix, thanks for your instant review :-)